onnx/defs/schema.cc

```diff
@@ -94,26 +94,39 @@ OpSchemaRegistry* OpSchemaRegistry::Instance() {
   return &instance;
 }
 
+std::string ConcatAllType(const std::unordered_set<std::string>& dt_set) {
+  std::stringstream ss;
+  for (const auto& dt: dt_set) {
+    ss << dt << ",";
+  }
+  return ss.str();
+}
+
 void OpSchema::CheckInputOutputType(struct InferenceContext& ctx) const {
-  std::unordered_map<std::string, DataType> type_constraints;
+  std::unordered_map<std::string, std::string> type_constraints;
```

Comment:
DataType and the related utils are used to use a pointer (to a string) guaranteed to be unique for a type so that pointer-comparison is sufficient to check for type-equality. Is there a need to use string instead of DataType here?

Reply:
No - it is just for debugging failed cases, will go back with pointers shortly.

Reply:
done.